aiopg/utils.py

```diff
@@ -231,3 +233,10 @@ def __exit__(self, *args):
         coroutines._COROUTINE_TYPES += (_ContextManager,)
     except:
         pass
+
+
+# map psycopg2cffi to psycopg2, so any code
+# that imports psycopg2 will use psycopg2cffi
+if PY_IMPL == 'PyPy':
+    from psycopg2cffi import compat
+    compat.register()
```

Comment:
@asvetlov do you mean you would prefer `psycopg2_compat` module with sth like this inside?

```python
# aiopg/psycopg2_compat

if PY_IMPL == 'PyPy':
    from psycopg2cffi import *
else:
    from psycopg2 import *
```